### PR TITLE
Azure CosmosDB for NoSQL

### PR DESCRIPTION
**Pull request by himanshu219**
## Purpose of this pull request

This pull request includes updates to collection steps and new section for install apps & dashboards.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/microsoft-azure/azure-cosmos-db.md [hunk 1]
```diff
@@ -18,12 +18,23 @@ The below instructions applies to the following [database APIs](https://learn.mi
 * Gremlin
 * Table
 
+:::note
+This app has been tested with MongoDB, Cassandra and NoSQL database account types.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-cosmos-db.md:22]:
```suggestion
This app is tested with MongoDB, Cassandra, and NoSQL database account types.
```

File: docs/integrations/microsoft-azure/azure-cosmos-db.md [hunk 2]
```diff
@@ -32,31 +41,157 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+<AppInstall2/>
+
+## Viewing the Azure CosmosDB for NoSQL dashboards
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+### Overview
+
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-cosmos-db.md:183]: Dashboard info needs to be added.

**Review by JV0812 — CHANGES_REQUESTED** [review]
@himanshu219 can you please add the dashboard info. Rest everything is fine.

**Comment by jpipkin1** [comment]
@himanshu219, I made some minor changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4714/commits/c426920094ee3313d9f86ddb12eff2ab731bedfa)").

File: docs/integrations/microsoft-azure/azure-cosmos-db.md [hunk 3]
```diff
@@ -32,31 +42,261 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+1. [Configure and deploy the ARM Template](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-2-configure-azure-resources-using-arm-template).
+1. [Export metrics to Event Hub](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-3-export-metrics-for-a-particular-resource-to-event-hub). Perform the following steps for each Azure Cosmos DB account that you want to monitor.
+   1. Choose **Stream to an event hub** as destination.
+   1. Select all the metrics under **Metrics** section.
+   1. Use the Event Hub namespace created by the ARM template in the previous step. You can create a new Event Hub or use the one created by the ARM template. You can use the default policy `RootManageSharedAccessKey` as the policy name.<br/><img src={useBaseUrl('img/integrations/microsoft-azure/azurecosmosdb-metrics.png')} alt="Azure Cosmos DB Diagnostic Settings for metrics" style={{border: '1px solid gray'}} width="800" />
+
+:::note
+Currently only Azure Cosmos DB for NoSQL database account type supports exporting metrics using diagnostic settings.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-cosmos-db.md:143]:
```suggestion
Currently, only Azure Cosmos DB for NoSQL database account type supports exporting metrics using diagnostic settings.
```

**Review by JV0812 — APPROVED** [review]
LGTM

**Pull request merged**